lib/lumberjack/client.rb

```diff
@@ -69,10 +67,9 @@ def initialize(opts={})
         :port => 0,
         :address => "127.0.0.1",
         :ssl_certificate => nil,
-        :window_size => 5000
       }.merge(opts)
       @host = @opts[:address]
-      @window_size = @opts[:window_size]
+      @window_size = 1
```

Comment:
I don't think I like this. I'd rather us move towards using Stud::Buffer or custom internal buffering to support retransmits better.